src/bech32.h

```diff
@@ -28,6 +28,14 @@ enum class Encoding {
     BECH32M, //!< Bech32m encoding as defined in BIP350
 };
 
+/** Character limits for bech32(m) encoded strings. Character limits are how we provide error location guarantees.
```

Comment:
nit: to be consistent with the casing below
```suggestion
/** Character limits for Bech32(m) encoded strings. Character limits are how we provide error location guarantees.
```

Reply:
Fixed.